lib/bundler/dsl.rb

```diff
@@ -290,25 +290,11 @@ def add_git_sources
         warn_deprecated_git_source(:github, <<-'RUBY'.strip, 'Change any "reponame" :github sources to "username/reponame".')
 "https://github.com/#{repo_name}.git"
         RUBY
-        # It would be better to use https instead of the git protocol, but this
-        # can break deployment of existing locked bundles when switching between
-        # different versions of Bundler. The change will be made in 2.0, which
-        # does not guarantee compatibility with the 1.x series.
-        #
-        # See https://github.com/bundler/bundler/pull/2569 for discussion
-        #
-        # This can be overridden by adding this code to your Gemfiles:
-        #
-        #   git_source(:github) do |repo_name|
-        #     repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
-        #     "https://github.com/#{repo_name}.git"
-        #   end
```

Comment:
I'm fine with removing this comment, since in my opinion it actually adds more confusion to the current situation...

Reply:
I agree. Just to confirm: no further action required here?

Reply:
Not from me!

I'll wait a bit to merge though in case there are other reviews or further discussion, specially since @segiddins approved this PR before I added my review.

Reply:
Actually... Can you fix the TravisCI build?

Reply:
Oh whoops! I didn't check how the build did. Should be fixed now, but I'll keep an eye on it.